.github/workflows/move-pr-to-in-review.yml

```
@@ -0,0 +1,23 @@
+
+on:
+  pull_request:
+    types: [review_requested]
+
+permissions:
+  pull-requests: write
+  contents: read
+
+jobs:
+  move-to-review:
+    if: ${{ secrets.UPDATE_PROJECT_V2_PAT && !github.event.pull_request.head.repo.fork }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)